Source/SwiftCloudant/HTTP/InterceptableSession/InterceptableSessionConfiguration.swift

```diff
@@ -0,0 +1,64 @@
+//
+//	File.swift
```

Comment:
Forgot to update 'File.swift'

Reply:
ahhhh, thanks for the catch 😆. will fix.